src/node/miner.cpp

```diff
@@ -106,7 +106,7 @@ void BlockAssembler::resetBlock()
     nFees = 0;
 }
 
-std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& scriptPubKeyIn)
+std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock()
```

Comment:
In commit "Drop script_pub_key arg from createNewBlock" (42f880aeb080070a42229d0bccd517100a3fe7fb)

Would it make sense to change the `BlockAssembler::CreateNewBlock()` signature in a separate followup commit? The test changes in this commit vastly outnumber the non-test changes, which I think make it more likely a bug in the non-test code could be missed.

Maybe this there could be two overloads initially:

```c++
std::unique_ptr<CBlockTemplate> CreateNewBlock()
{
    return CreateNewBlock(m_options.coinbase_output_script ? *m_options.coinbase_output_script : CScript{} << OP_TRUE);
}

std::unique_ptr<CBlockTemplate> CreateNewBlock(const CScript& scriptPubKeyIn);
```

So test code in could updated in a separate the commit and then the overload could be deleted.

Reply:
I made an overload, though slightly differently.